package.json

```diff
@@ -7,10 +7,11 @@
     "db:migrate": "npx @better-auth/cli migrate --yes",
     "prettier:check": "prettier --check '**/*.{js,css,md,yml}'",
     "prettier:write": "prettier --write '**/*.{js,css,md,yml}'",
-    "prepare": "husky",
+    "husky": "husky",
     "lint": "eslint ."
   },
   "dependencies": {
+    "nodemon": "^3.1.10",
```

Comment:
Why do we need nodemon to run the service?
We can start it with `node src/index.js`, or if we're feeling fancy use [pm2](https://www.npmjs.com/package/pm2) to keep it running forever and across configuration changes.

I think for the use case of running it in a container, we should use `node` and have the container restart, if the process fails its health checks

Reply:
I rolled it back. Didn't realize I could just use node. That's better.